src/execution_plan/ops/op_expand_into.c

```diff
@@ -147,7 +147,8 @@ static Record _handoff(OpExpandInto *op) {
 		// Current record resides at row recordCount.
 		int rowIdx = op->recordCount;
 		op->r = op->records[op->recordCount];
-
+		assert(Record_GetType(op->r, op->srcNodeIdx) == REC_TYPE_NODE);
+		assert(Record_GetType(op->r, op->destNodeIdx) == REC_TYPE_NODE);
```

Comment:
Consider moving this assertion into `Record_GetNode` if possible.
```
Node *Record_GetNode(const Record r, int idx) {
	r->entries[idx].type = REC_TYPE_NODE;
	return &(r->entries[idx].value.n);
}
```
Not sure if setting the entry type here is still needed, or in which situation the type is actually changed.

Reply:
Create ops use `Record_GetNode` to access zeroed memory which they then populate.

I don't really love this approach, but don't want to rewrite it in this PR!